### ISSUE TITLE
Some concepts have disappeared 

### ISSUE THREAD

**Issue by cpommier** [MIAPPE/MIAPPE-ontology#2]
@LlorencCB
The following concepts where used by PHIS and are no longer available. 
Quality : environment feature & trait 
http://purl.org/ppeo/PPEO.owl#PPEO_000010
http://purl.org/ppeo/PPEO.owl#PPEO_000009
http://purl.org/ppeo/PPEO.owl#PPEO_000008

**Comment by cpommier** [comment]
```
-------- 
  <owl:ObjectProperty rdf:about="http://purl.org/ppeo/PPEO.owl#hasQuality">
        <rdfs:domain rdf:resource="http://purl.org/ppeo/PPEO.owl#PPEO_000007"/>
        <rdfs:range rdf:resource="http://purl.org/ppeo/PPEO.owl#PPEO_000008"/>
        <rdfs:label xml:lang="en">has quality</rdfs:label>
    </owl:ObjectProperty>
----------
 <owl:Class rdf:about="http://purl.org/ppeo/PPEO.owl#PPEO_000008">
        <rdfs:label xml:lang="en">quality</rdfs:label>
    </owl:Class>

    <owl:Class rdf:about="http://purl.org/ppeo/PPEO.owl#PPEO_000009">
        <rdfs:subClassOf rdf:resource="http://purl.org/ppeo/PPEO.owl#PPEO_000008"/>
        <rdfs:label xml:lang="en">trait</rdfs:label>
    </owl:Class>

<owl:Class rdf:about="http://purl.org/ppeo/PPEO.owl#PPEO_000007">
       ...
                <owl:onProperty rdf:resource="http://purl.org/ppeo/PPEO.owl#hasMethod"/>
            ...
                <owl:onProperty rdf:resource="http://purl.org/ppeo/PPEO.owl#hasQuality"/>
              ...
                <owl:onClass rdf:resource="http://purl.org/ppeo/PPEO.owl#PPEO_000008"/>
        ...
                <owl:onProperty rdf:resource="http://purl.org/ppeo/PPEO.owl#hasScale"/>
             ...
        <rdfs:label xml:lang="en">observation variable</rdfs:label>
    </owl:Class>


------
    <owl:Class rdf:about="http://purl.org/ppeo/PPEO.owl#PPEO_000010">
        <rdfs:subClassOf rdf:resource="http://purl.org/ppeo/PPEO.owl#PPEO_000008"/>
        <rdfs:label xml:lang="en">environment feature</rdfs:label>
    </owl:Class>
```

**Comment by cpommier** [comment]
The environment part has been heavily rewritten. According to the old schemas (https://drive.google.com/file/d/1wK9KR-D2hWnaoc94pPA8GZzjAbw8jtYr/view) the part @LlorencCB is missing is about the trait/method/scale of the observation variables, and more precisely about the trait (aka quality).
@LlorencCB I would recommend switching to trait (which can be either phenotyping or environment trait)
http://purl.org/ppeo/PPEO.owl#trait

Let me know if that works for you and close the issue if it is solved ?

**Comment by AgreementMakerLight** [comment]
I think the main issue here is that when we updated PPEO, we decided to overhaul the URIs of the classes from alpha-numeric codes (e.g. PPEO_000009) to textual names (e.g. trait) in the URIs.
We were already using proper names for the properties, so it was a bit weird to have a different standard for classes and properties. More importantly, we wanted to use PPEO to express BrAPI in RDF, and having textual class names makes RDF a lot easier for human interpretation.
This means that every URI that PHIS was importing from PPEO needs to be revised to point to the corresponding new textual URI.
Of the classes you listed, PPEO_000009 is still there with the same usage, under the name "trait", but PPEO_000008 and PPEO_000010 were removed, as we now treat "environment features" that are measured throughout the experiment as traits (so there is no need to separate them or to have a grouping concept above them).
There is, however, still the concepts "environment" and "environment_parameter" which refer to aspects that were kept approximately constant throughout the experiment. Depending on the intended usage in PHIS, it may still make sense to point to these concepts instead of PPEO_000010.
-- Daniel
